### DISCUSSION TITLE
How to get all `RanExperiment`s?

### DISCUSSION THREAD

**Discussion by jacob-danner** [Arize-ai/phoenix#3993] [Q&A] [answered]
I've been exploring datasets and experiments. My team needs to be able to get the data out of each experiment. 

I've found the `get_evaluations` method:

```
experiment = run_experiment(
    golden_dataset,
    task,
    experiment_name="task_experiment",
    evaluators=[some_evaluator]
)

# type(experiment) => phoenix.experiments.types.RanExperiment

experiment.get_evaluations()
```

This seems to do the trick for one experiment. However, I need to be able to get the data of _all_ of my experiments. 

My question is: how can I access the list of all `RanExperiment`s? 

I'm imagining some function like `get_all_experiments`, must exist, but I can't find it in the code base. Being able to access such a list would enable use of the `get_evaluations` method to collect all data across ran experiments.

Thanks!

**Comment by axiomofjoy** [comment, marked as answer]
Thanks @jacob-danner. At the moment, we don't have a list experiments client method. I have created an enhancement request [here](https://github.com/Arize-ai/phoenix/issues/3994).

**Reply by jacob-danner** [reply]
Awesome, thanks for the speedy reply 👍

**Reply by mikeldking** [reply]
Thanks for the feedback @jacob-danner happy to iterate on any asks you may have!
